### PR TITLE
Athipp dev3

### PR DESCRIPTION
**Pull request by athipp**
Adding dataplane APIs for Azure Analysis Services . This was originally reviewed as PR https://github.com/Azure/azure-powershell/pull/3151 but then that PR got dirty so submitting another PR. Changes are : 
1. Dataplane APIs.
2. For AnalysisServices, updated the version from 0.0.1 to 0.0.2
3. Added a changelog.md

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md).**
- [ ] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/dev/documentation/platyPSHelp-documentation.md).**
- [ ] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.
- [ ] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [ ] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://gist.github.com/markcowl/338e16fe5c8bbf195aff9f8af0db585d#what-is-the-change).
- [ ] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThrough` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [ ] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [ ] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [ ] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

### CODE REVIEW

File: src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.psd1 [hunk 1]
```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '0.0.1'
+ModuleVersion = '0.0.2'
```
Comment (shahabhijeet) [inline src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.psd1]: We take care of this, you should not be changing any of this.

File: src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.psd1 [hunk 2]
```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '0.0.1'
```
Comment (shahabhijeet) [inline src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.psd1]: This PR is targeting dev branch, it should be targeting release 3.2.0 branch. Please send a new PR targeting the right branch.

**Comment by shahabhijeet** [comment]
@athipp this PR still has the same issue as the previous where random files are getting merged in this PR. Please have a PR that has only your files/changes.

File: src/ResourceManager/AnalysisServices/ChangeLog.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Current Release
+
+## Version 0.0.2
```
Comment (athipp) [inline src/ResourceManager/AnalysisServices/ChangeLog.md]: done.

File: src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.psd1 (hunk 1 above)
Comment (athipp) [inline src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.psd1]: done. Will leave it as is

File: src/ResourceManager/AnalysisServices/ChangeLog.md (hunk 3 above)
Comment (athipp) [inline src/ResourceManager/AnalysisServices/ChangeLog.md]: Done.

File: src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.psd1 (hunk 2 above)
Comment (athipp) [inline src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.psd1]: Done

**Comment by athipp** [comment]
Will submit a new one. not sure how these files are getting picked up.

**Pull request closed without merging**